junit5/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>org.jboss.weld</groupId>
+        <artifactId>weld-junit-parent</artifactId>
+        <version>1.1.1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>weld-junit5</artifactId>
+    
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <!--We don't have JUnit 5 deps anywhere else yet.-->
+        <junit.platform.version>1.0.1</junit.platform.version>
+        <junit.jupiter.version>5.0.1</junit.jupiter.version>
```

Comment:
Junit5 versions should be defined in `weld-junit-parent` properties and the dependencies should be managed in its `<dependencyManagement>`.